src/main/java/edu/wpi/grip/ui/preview/SocketPreviewView.java

```diff
@@ -40,22 +32,7 @@ public SocketPreviewView(EventBus eventBus, Socket<T> socket) {
         this.setCollapsible(false);
     }
 
-    public Socket<T> getSocket() {
+    public OutputSocket<T> getSocket() {
```

Comment:
Everywhere else it's called `getSocket()` unless a class needs to distinguish between the two (like `Connection`).  Also, calling it that sort of implies that this outputs something, which it doesn't.